tracardi/process_engine/action/v1/contains_string_action.py

```diff
@@ -0,0 +1,102 @@
+from pydantic import validator
+
+from tracardi.service.plugin.domain.config import PluginConfig
+
+from tracardi.service.plugin.domain.register import Plugin, Spec, MetaData, Documentation, PortDoc, Form, FormGroup, \
+    FormField, FormComponent
+from tracardi.service.plugin.runner import ActionRunner
+from tracardi.service.plugin.domain.result import Result
+
+
+class Config(PluginConfig):
```

Comment:
I'd move it to another directory, should be somewhere in operations or strings section of /action/v1, with config separated in .mode.config file